packages/ng/src/app/select/option/select-option.module.ts

```diff
@@ -1,16 +1,22 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { LuSelectOption } from './select-option.component';
+import { LuOptionFeederDirective } from './select-option-feeder.directive';
+import { AbstractSelectOptionFeederComponent } from './select-option-feeder.component';
 
 @NgModule({
 	imports: [
 		CommonModule,
 	],
 	declarations: [
 		LuSelectOption,
+		LuOptionFeederDirective,
+		AbstractSelectOptionFeederComponent,
```

Comment:
no need to declare Abstract components as they eil _never_ be initialised, otherwise they're not abstract

Reply:
I use the term Abstract because, i really want to mutualize code and act like an Abstract class but you cannot create in angular real Abstract Components... So the best I found is to specify in the name that it is an "Abstract class"

Reply:
you cant specify abstract component thats true, but you can create an abstract class and extend it in a concrete component

ie

```ts
ezxport abstract class ASelectOptionFeeder { ... }
@Component({...})
export class SelectOptionFeederComponent exstends ASelectOptionFeeder {...}
```

Reply:
good point !

Reply:
`export abstract class ASelectOptionFeeder<T extends ISelectOption = ISelectOption>` if you wann go full generics